addons/mail/static/src/views/fields/assign_user_command_hook.js

```diff
@@ -63,13 +63,18 @@ export function useAssignUserCommand() {
                 domain = Domain.and([domain, [["id", "not in", selectedUserIds]]]);
             }
         }
-        const searchResult = await orm.call(component.props.relation, "name_search", [], {
+        if (component._pendingRpc) {
+            component._pendingRpc.abort();
+        }
+        component._pendingRpc = orm.call(component.props.relation, "name_search", [], {
             name: value,
             args: domain.toList(),
             operator: "ilike",
             limit: 80,
             context,
         });
+        const searchResult = await component._pendingRpc;
+        component._pendingRpc = null;
```

Comment:
`.abort()` does reject the promise, which could lead to traceback "Uncaught Promise Error".
Apparently passing positional argument `false` allows to not throw an error:

```js
component._pending?.abort(false);
```

Reply:
Tnx for pointing this out! After reading the definition, indeed it throws an error (in my case it was handled higher in the call stack I guess so I didn't notice it during testing). https://github.com/odoo/odoo/blob/4288838ecdd059c3c0780df6e06ad9e2c83eb675/addons/web/static/src/core/network/rpc_service.js#L102-L118

As for the optional chaining, I've looked up my notes and apparently it was only allowed since saas-16.1, so for this version I've left it with the explicit `if` case. 😃